<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RayView_rtiow</class>
 <widget class="QDialog" name="RayView_rtiow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>929</width>
    <height>638</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="m_tempsTotal">
       <property name="text">
        <string>text</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="m_tempsProcess">
       <property name="text">
        <string>text</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="m_tempsImage">
       <property name="text">
        <string>text</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGraphicsView" name="graphicsView"/>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item row="3" column="1">
      <widget class="QSlider" name="horizontalSlider_4">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>500</number>
       </property>
       <property name="pageStep">
        <number>1</number>
       </property>
       <property name="value">
        <number>1</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TicksAbove</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>SphereY</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QDoubleSpinBox" name="dSpin1">
       <property name="minimum">
        <double>-99.989999999999995</double>
       </property>
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
       <property name="value">
        <double>0.500000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QDoubleSpinBox" name="dSpin2">
       <property name="minimum">
        <double>-99.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Rien</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSlider" name="horizontalSlider">
       <property name="minimum">
        <number>-100</number>
       </property>
       <property name="maximum">
        <number>500</number>
       </property>
       <property name="value">
        <number>30</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TicksAbove</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSlider" name="horizontalSlider_3">
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>10</number>
       </property>
       <property name="tracking">
        <bool>true</bool>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="invertedAppearance">
        <bool>false</bool>
       </property>
       <property name="invertedControls">
        <bool>false</bool>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TicksAbove</enum>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QCheckBox" name="chkColor">
       <property name="text">
        <string>Color</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSlider" name="horizontalSlider_2">
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>30</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TicksAbove</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Depth</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>NumSamples</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="btnGo">
       <property name="text">
        <string>Go</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>RayView_rtiow</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>RayView_rtiow</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
